src/NoValue.php

```diff
@@ -5,6 +5,9 @@
 final class NoValue
 {
 
+	/**
+	 * Represents a completely missing value
+	 */
```

Comment:
I meant for the function body, to make coding standard happy

Reply:
Yeah, should be ok now :D